src/containers/alerts.jsx

```diff
@@ -0,0 +1,25 @@
+import {connect} from 'react-redux';
+
+import {
+    showAlert,
+    closeAlert,
+    onReconnect
+} from '../reducers/alerts';
+
+import AlertsComponent from '../components/alerts/alerts.jsx';
+
+const mapStateToProps = state => ({
+    visible: state.scratchGui.alerts.visible,
+    message: state.scratchGui.alerts.message
+});
+
+const mapDispatchToProps = dispatch => ({
+    onShowAlert: () => dispatch(showAlert()),
+    onCloseAlert: () => dispatch(closeAlert()),
+    onReconnect: () => dispatch(onReconnect())
```

Comment:
Looking at this more, I don't think this is actually used anywhere, so it can be removed altogether.